src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicMethodExtractor.VisualBasicCodeGenerator.vb

```diff
@@ -52,16 +50,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.ExtractMethod
                 Me._methodName = CreateMethodName().WithAdditionalAnnotations(MethodNameAnnotation)
             End Sub
 
+            Protected Overrides Function UpdateMethodAfterGenerationAsync(originalDocument As SemanticDocument, methodSymbolResult As OperationStatus(Of IMethodSymbol), cancellationToken As CancellationToken) As Task(Of SemanticDocument)
+                Return Task.FromResult(originalDocument)
+            End Function
```

Comment:
base method became abstract to simplify things.  so this is just that impl inlined here.